core/src/settings.ts

```diff
@@ -44,6 +44,8 @@ export const MAX_DATA_STALENESS = 5_000
 export const REMOVE_ON_COMPLETE = 500
 export const REMOVE_ON_FAIL = 1_000
 export const CONCURRENCY = 12
```

Comment:
I think later on we could also add optinal ENV Variable for  bullmq `CONCURRENCY` variable.

Reply:
Good point! Later, I think we should split the `CONCURRENCY` to more  specific variables and expose it as the `DATA_FEED_REPORTER_CONCURRENCY`.